helm/validator/values.yaml

```diff
@@ -0,0 +1,366 @@
+chain:
+  name: premainnet
```

Comment:
is the idea to have premainnet publicly available to everyone?

Reply:
No, it will just serve as an example configuration. Without genesis blob and waypoint no one can connect to any existing chain

Reply:
understood